### ISSUE TITLE
How to deal with depth = 0 ?

### ISSUE THREAD

**Issue by Fangyh09** [dontLoveBugs/DORN_pytorch#23]
Hello,
when depth is zero, then log(depth) is illegal. How to deal with it ?
https://github.com/dontLoveBugs/DORN_pytorch/blob/4fed3629282849af66f2d7fff12e12a06fb68b7e/utils.py#L91-L115

**Comment by LiZhenLiangLee** [comment]
@Fangyh09 
Hi, I think you can find the answer in Section 3.2 of this paper.
They add a shift sigma to both alpha and beta to apply SID .

**Comment by dontLoveBugs** [comment]
You need add a valid mask(gt > 0.) to make all ground truth values are valid, so torch.log(depth) is  illegal but doesn't make a difference.
